### ISSUE TITLE
WCAG 1.4.3: Elements must have sufficient color contrast (.text)

### ISSUE THREAD

**Issue by jwunderl** [microsoft/pxt-microbit#2295]
#### Issue
`Elements must have sufficient color contrast` ([color-contrast](https://dequeuniversity.com/rules/axe/3.2/color-contrast?application=msftAI))
#### Target application
[Microsoft MakeCode for micro:bit](https://makecode.microbit.org/#editor)
#### Element path
.secondary > .text
#### Snippet
`<span class="ui text">Experiments</span>`
#### How to fix
    Fix any of the following:
      Element has insufficient color contrast of 2.55 (foreground color: #ffffff, background color: #95a5a6, font size: 12.0pt, font weight: bold). Expected contrast ratio of 4.5:1
#### Environment
Chrome version 75.0.3770.142
This accessibility issue was found using Accessibility Insights for Web 2.7.1 (axe-core 3.2.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).

experiment button light gray with white text

![Screen Shot 2019-07-18 at 2 58 46 PM](https://user-images.githubusercontent.com/5615930/61494994-930a5200-a96c-11e9-9616-7c2d1156de59.png)

**Comment by BhuwanNaswa** [comment]
Labels
#A11yMAS, #HCL, #HCL-MakeCode-E2E, #MAS1.4.3, #Win10-Chrome, #Win10-Edge, #accessibility, #K4W, #closed
